apps/dsp-app/src/app/user/user-form/user-form.component.html

```diff
@@ -1,92 +1,73 @@
-<!-- content -->
-<dasch-swiss-app-progress-indicator *ngIf="loadingData"></dasch-swiss-app-progress-indicator>
-<div *ngIf="!loadingData">
-  <form [formGroup]="userForm" class="form-content user-data" (ngSubmit)="submitData()">
-    <!-- username -->
-    <mat-form-field [class.large-field]="!user" [class.small-field]="user">
-      <input
-        matInput
-        type="text"
-        formControlName="username"
-        placeholder="{{ 'appLabels.form.user.general.username' | translate }}" />
-      <mat-hint *ngIf="formErrors.username"> {{ formErrors.username }} </mat-hint>
-    </mat-form-field>
+<form [formGroup]="userForm" class="form-content user-data">
+  <mat-form-field class="{{editExistingUser ? 'large-field' : 'small-field'}}">
```

Comment:
use [class.]

Reply:
Oh, FYI: I completely disregarded any css and kept everything as it was - because the css will be changed later anyway and I like to have that separated. We should address that later.